frontend/pages/organizations/index.vue

```diff
@@ -24,21 +24,15 @@
 </template>
 
 <script setup lang="ts">
-import { Organization } from "../../types/organization";
 
 definePageMeta({
   layout: "sidebar",
 });
-
-const organization: Organization = {
-  name: "tech from below",
-  status: "approved",
-  tagline: "Technologie von und für soziale Bewegungen",
-  location: "Berlin",
-  description: "This is the description of tech from below.",
-  topic: "Technology and Privacy",
-  members: 3,
-  supporters: 30,
-  imageURL: "/images/tech-from-below.svg",
-};
+const sidebar = useSidebar();
```

Comment:
Am gonna check whether this is necessary. Was randomly on some pages and wasn't needed.